scripts/run_e2e_tests.sh

```diff
@@ -24,8 +24,10 @@ fi
 # Create a new namespace for e2e tests
 oc new-project $E2E_TEST_NS
 
+export ARGOCD_CLUSTER_CONFIG_NAMESPACES=openshift-gitops
```

Comment:
AFAIU the value of key `ARGOCD_CLUSTER_CONFIG_NAMESPACES` is already set to `openshift-gitops` in https://github.com/redhat-developer/gitops-operator/blob/e9e2db1cc42006e1b24d663d5ce5a66785f826b1/deploy/operator.yaml#L33. Do we need it again ?

Reply:
We do: when running `run_e2e_tests.sh` that operator.yaml isn't used, so it's necessary to set the environment variable in `run_e2e_tests.sh`.

Reply:
Make sense. Thanks!